cppcache/test/LoggingTest.cpp

```diff
@@ -103,7 +103,7 @@ class LoggingTest : public testing::Test {
                                  const char* filename, int32_t rollIndex) {
     auto baseName = boost::filesystem::path(filename).stem().string();
     auto rolledPath =
-        logdir / boost::filesystem::path(baseName + "-" +
+        logdir / boost::filesystem::path(baseName + "." +
```

Comment:
By nature, any test that using the filesystem can't be a unit test. This test should be moved to integration/test or use some in-memory log sink.

Reply:
Integration tests should only be using the public API and dynamically linking to geode-native.  The logger is not a public API.  I'm unsure how I would go about testing file size and disk space limits using an in-memory sink - did you have something in mind?